cache.rst

```diff
@@ -304,6 +306,83 @@ For advanced configurations it could sometimes be useful to use a pool as an ada
 
 
 
+Custom Provider Options
+-----------------------
+
+Some providers have specific options that could be configured. The
+:doc:`RedisAdapter </components/cache/adapters/redis_adapter>` allows you to create
+providers with option ``lazy``, ``timeout`` etc. To use these options with non-default
```

Comment:
all this section is correct, but one should know that "lazy" can also be passed to the DSN `?lazy=true`, see `RedisTrait::$defaultConnectionOptions`

Reply:
there aren't any options that you cannot set using a DSN I think :)

Reply:
I tried find something that was supported in 3.4. I will use `retry_interval`.

Thank you for the feedback.